extensions/wikia/CreateNewWiki/tasks/CreateDatabase.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Wikia\CreateNewWiki\Tasks;
+
+class CreateDatabase implements Task {
+
+	use \Wikia\Logger\Loggable;
+
+	//TODO would be awesome to read it from some config
+	const ACTIVE_CLUSTER = "c7";
+
+	/** @var  TaskContext */
+	private $taskContext;
+
+	/** @var  string */
+	private $clusterDB;
+
+	public function __construct( $taskContext ) {
+		$this->taskContext = $taskContext;
+	}
+
+	protected function getLoggerContext() {
```

Comment:
This actually causes the loggable trait to append this information to all log invocation (you then don't need to pass it in the context field in each log invocation, which is easy to forget about)